internal/component/database_observability/mysql/collector/connection_info.go

```diff
@@ -36,13 +38,14 @@ func NewConnectionInfo(args ConnectionInfoArguments) (*ConnectionInfo, error) {
 		Namespace: "database_observability",
 		Name:      "connection_info",
 		Help:      "Information about the connection",
-	}, []string{"provider_name", "provider_region", "db_instance_identifier", "engine"})
+	}, []string{"provider_name", "provider_region", "db_instance_identifier", "engine", "db_version"})
```

Comment:
Change seems ok to me, I'd just check for opinions on the label name. Maybe given that we're using `engine` terminology already, this new one could be `engine_version`?

And maybe we should also rename the `engine` to `engine_name`?

Reply:
Renamed the new label to `engine_version`. I think `engine` is fine, and self explanatory, and changing it now would have non trivial downstream impacts. Happy to revisit if we think it needs the additional clarity,